packages/tpetra/core/src/Tpetra_FECrsGraph_decl.hpp

```diff
@@ -172,8 +207,43 @@ namespace Tpetra {
                 const Teuchos::RCP<const map_type> & ownedPlusSharedRowMap,
                 const Kokkos::DualView<const size_t*, execution_space>& numEntPerRow,
                 const Teuchos::RCP<const import_type> & ownedPlusSharedToOwnedimporter = Teuchos::null,
-                const Teuchos::RCP<const map_type> & domainMap = Teuchos::null,
-                const Teuchos::RCP<const map_type> & rangeMap = Teuchos::null,
+                const Teuchos::RCP<const map_type> & ownedDomainMap = Teuchos::null,
+                const Teuchos::RCP<const map_type> & ownedRangeMap = Teuchos::null,
+                const Teuchos::RCP<Teuchos::ParameterList>& params = Teuchos::null);
+
+    /// \brief Constructor for globally-indexed assembly specifying a (possibly different) upper
```

Comment:
Please add documentation saying why one would use this constructor over the existing constructors.